### PR TITLE
api initialized for User and Ticket models

### PR DESCRIPTION
**Pull request by ethanmnrd**
use 'python manage.py runserver' within /server/ to check out http://127.0.0.1:8000/api/etes/ and http://127.0.0.1:8000/api/etes/ticket

### CODE REVIEW

File: server/etes/urls.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+urlpatterns = [
+    path('api/etes/', views.UserListCreate.as_view() ),
+    path('api/etes/ticket', views.TicketListCreate.as_view() ),
+]
```
Comment (jooohhn) [inline server/etes/urls.py]: Fix new line problem here

File: server/etes/serializers.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = '__all__'
+
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = '__all__'
```
Comment (jooohhn) [inline server/etes/serializers.py]: Fix new line problem here

File: server/etes/views.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+from rest_framework import generics
 
 # Create your views here.
+class UserListCreate(generics.ListCreateAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+class TicketListCreate(generics.ListCreateAPIView):
+    queryset = Ticket.objects.all()
+    serializer_class = TicketSerializer
```
Comment (jooohhn) [inline server/etes/views.py]: Fix new line problem here

File: server/etes/urls.py (hunk 1 above)
Comment (ethanmnrd) [reply server/etes/urls.py]: Resolved.

File: server/etes/serializers.py (hunk 2 above)
Comment (ethanmnrd) [reply server/etes/serializers.py]: Resolved.

File: server/etes/views.py (hunk 3 above)
Comment (ethanmnrd) [reply server/etes/views.py]: Resolved.

**Pull request merged**